docs/SPIR-V.rst

```diff
@@ -299,6 +299,7 @@ Supported extensions
 * SPV_GOOGLE_hlsl_functionality1
 * SPV_GOOGLE_user_type
 * SPV_NV_mesh_shader
+* SPV_AMD_shader_early_and_late_fragment_tests
```

Comment:
Could you sort this list? This should make it easier to scan for specific extensions